products/statement-generator/src/components-layout/AppFooter.tsx

```diff
@@ -91,6 +100,14 @@ const useStyles = makeStyles(({ breakpoints, palette, spacing }) =>
       margin: 'auto 0',
       fontWeight: 300,
 
+      [breakpoints.between('md', 'lg')]: {
+        display: 'flex',
+      },
+
+      [breakpoints.between('xs', 'sm')]: {
+        display: 'none',
+      },
+
       [breakpoints.down(breakpoints.values.sm)]: {
         display: 'none',
       },
```

Comment:
these also look to do the same thing.

Reply:
these actually do not do the same thing. one is for the line between the links  the other is for the right column. `.between` is for screen sizes between two sizes, `.down` is for small and under.  if i remove .between for the line breakpoints the lines show up in a column like so: 
![Screenshot 2024-01-31 at 9 07 45 PM](https://github.com/hackforla/expunge-assist/assets/117246204/eb899a0d-62f2-4278-b6c3-ed2adf8b0937)